src/core/AnimatedValue.js

```diff
@@ -14,9 +16,25 @@ export default class AnimatedValue extends AnimatedNode {
   }
 
   __detach() {
+    this.__detachAnimation(false);
```

Comment:
Same comment here as in `setValue()` – it appears this won't actually stop the animation from running, we will just invoke the callback

Reply:
same